android/app/src/main/java/app/organicmaps/car/util/RoutingUtils.java

```diff
@@ -93,20 +92,13 @@ private static Step createNextStep(@NonNull final CarContext context, @NonNull R
     return builder.build();
   }
 
+  @SuppressWarnings("NewApi") // ZonedDateTime is backported for Android versions below 8.0.
```

Comment:
Why linter is not aware of desugaring?

Reply:
That's not for the linter, it's for the builder constructor (it's marked as requiring API level 26 due to the `ZonedDateTime` parameter).